examples/main/main.cpp

```diff
@@ -82,6 +82,14 @@ struct whisper_params {
 
 void whisper_print_usage(int argc, char ** argv, const whisper_params & params);
 
+char* whisper_param_turn_lowercase(char* in){
+    int string_len = strlen(in);
+    for(int i = 0; i < string_len; i++){
+        *(in+i) = tolower(*(in+i));
```

Comment:
```suggestion
        *(in+i) = tolower((unsigned char)*(in+i));
```

https://en.cppreference.com/w/cpp/string/byte/tolower